src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -610,6 +613,12 @@ public void Remove(Page page)
             {
                 navigationPages[PageCount - 2].Opacity = 1.0f;
                 navigationPages[PageCount - 2].SetVisible(true);
+                NotifyAccessibilityStatesChangeOfPages(page, navigationPages[PageCount - 2]);
+            }
+ 
+	    if ((page == Peek()) && (PageCount == 1))
```

Comment:
IMO we could reduce duplication as below
if (page == Peek())
{
    if (PageCount >= 2)
    {
                navigationPages[PageCount - 2].Opacity = 1.0f;
                navigationPages[PageCount - 2].SetVisible(true);
                NotifyAccessibilityStatesChangeOfPages(page, navigationPages[PageCount - 2]);
    }
    else if (PageCount == 1)
    {
        NotifyAccessibilityStatesChangeOfPages(page, null);
    } 
}

Reply:
Yes, you are right.